clients/imodels-client-management/src/base/internal/OperationsBase.ts

```diff
@@ -3,20 +3,23 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import { Constants } from "../../Constants";
-import { AuthorizationParam, BinaryContentType, ContentType, Dictionary, PreferReturn, RestClient, SupportedGetResponseTypes } from "../types";
+import { AuthorizationParam, BinaryContentType, ContentType, Dictionary, HeaderFactories, HeadersParam, PreferReturn, RestClient, SupportedGetResponseTypes } from "../types";
 
 import { CollectionResponse } from "./ApiResponseInterfaces";
 import { EntityCollectionPage } from "./UtilityTypes";
 
-type SendGetRequestParams = AuthorizationParam & { url: string, preferReturn?: PreferReturn, responseType?: SupportedGetResponseTypes };
-type SendPostRequestParams = AuthorizationParam & { url: string, body: object | undefined };
-type SendPutRequestParams = AuthorizationParam & { url: string, contentType: BinaryContentType, body: Uint8Array };
-type SendPatchRequestParams = SendPostRequestParams;
-type SendDeleteRequestParams = AuthorizationParam & { url: string };
+interface CommonRequestParams extends AuthorizationParam, HeadersParam {
```

Comment:
I think this would look prettier as a type, then the empty `{}` would not be needed.